Makefile

```diff
@@ -54,6 +54,19 @@ docker-build: test
 docker-push:
 	docker push ${IMG}
 
+kind-e2e:
```

Comment:
you'd better add a github action to make this e2e test trigger on every PR. 
https://github.com/oam-dev/trait-injector/blob/master/.github/workflows/go.yml